source/sass/buyers-guide/views/product.scss

```diff
@@ -84,6 +84,14 @@
   h4 {
     font-family: "Nunito Sans";
   }
+
+  @import "../creep-face-emoji";
+
+  .creep-face-emoji {
+    top: 3px;
+    margin: 0;
+    margin-right: 0.25em;
```

Comment:
the 2 margin rules can be merged into `margin: 0 0.25em 0;`

Reply:
they can, but it's always seemed weird to me to list "0" three times when there's a way to set all of them at once and then only add your exception. Do we have a preferred format?

Reply:
Yes. We shouldn't be overriding rules within the same declaration block. So in this case we should only have one rule that deals with `margin`